Dependecy Error in rsuite@"^4.9.3" -> @rsuite/icons@1.0.0-alpha.3 for React v17

### What version of rsuite are you using?

"rsuite": "^4.9.3",

### What version of React are you using?

"react": "^17.0.2",

### What browser are you using?

Chrome

### Describe the Bug

After upgrade from "rsuite": "^4.8.9" to "rsuite": "^4.9.3", i receive this error, in rsuite 4.8.9 works perfectly

```
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from @rsuite/icons@1.0.0-alpha.3
npm ERR! node_modules/rsuite-table/node_modules/@rsuite/icons
npm ERR!   @rsuite/icons@"^1.0.0-12" from rsuite-table@3.14.5
npm ERR!   node_modules/rsuite-table
npm ERR!     rsuite-table@"^3.14.4" from rsuite@4.9.3
npm ERR!     node_modules/rsuite
npm ERR!       rsuite@"^4.9.3" from the root project
npm ERR!       1 more (@rsuite/responsive-nav)
```

### Expected Behavior

install normally, the downgrade is not possible because I already have other libraries working with React v17.

--- a/package.json
+++ b/package.json
@@ -63,7 +63,7 @@
     "react-lifecycles-compat": "^3.0.4",
     "react-virtualized": "^9.21.0",
     "recompose": "^0.30.0",
-    "rsuite-table": "^3.14.4",
+    "rsuite-table": "^3.14.6",
     "schema-typed": "^1.5.1"
   },
   "peerDependencies": {

